foundationsync/indico_foundationsync/sync.py

```diff
@@ -125,10 +131,35 @@ def _update_room(self, room, room_data):
         room.is_deleted = False
         room.is_reservable = True
         for k, v in room_data.iteritems():
+            if k == 'manager_group':
+                continue
             if getattr(room, k) != v:
                 setattr(room, k, v)
         db.session.flush()
 
+    def _update_managers(self, room, room_data, room_role_map):
+        new_managers = {room.owner}
+        manager_group = room_data.get('manager_group')
+
+        # add managers from e-group
+        if manager_group is not None:
+            group = GroupProxy(manager_group.strip(), provider='cern-ldap')
+            if group.group is None:
+                self._logger.warning("Group '%s' does not exist in LDAP", manager_group)
+            new_managers.add(group)
+
+        # add managers from aisroles (DKMs + DKAs)
+        new_managers |= {get_user_by_email(email, create_pending=True)
+                         for email in room_role_map[(room.building, room.floor, room.number)]}
```

Comment:
can we end up with a room with no entry in the role map?

Reply:
No, since the owner will always be there (otherwise we won't update it).